.travis.yml

```diff
@@ -3,4 +3,7 @@ cache: bundler
 sudo: false
 
 rvm:
-  - 2.3.1
+  - '2.2'
```

Comment:
I seem to recall we removed support for ruby 2.2 a while back.

Reply:
Thanks, ok I remove it.